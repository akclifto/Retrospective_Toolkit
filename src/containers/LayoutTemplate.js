import React from 'react';
import Header from '../components/Header';
import Footer from '../components/Footer';
import { makeStyles } from '@material-ui/core';

const useStyles = makeStyles(() => ({

    template: {
        minHeight: '100vh',
    }
}));

const LayoutTemplate = (props) => {
    const classes = useStyles();

        return (
            <div className={classes.template}>
                    <Header />
                        <div>
                            {props.children} 
                        </div>
                    <Footer />          
            </div>
        );
}
export default LayoutTemplate;