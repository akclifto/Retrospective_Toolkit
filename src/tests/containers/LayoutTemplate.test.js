import React from "react";
import { shallow } from "enzyme";
import LayoutTemplate from "../../containers/LayoutTemplate";

describe("Container/DiceLanding Testing", () => {
  // Snapshot for LandingPage
  test("SNAPSHOT: diceLanding", () => {
    const wrapper = shallow(<LayoutTemplate/>);
    expect(wrapper).toMatchSnapshot();
  });
});