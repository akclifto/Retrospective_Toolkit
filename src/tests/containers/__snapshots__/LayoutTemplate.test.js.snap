// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container/DiceLanding Testing SNAPSHOT: diceLanding 1`] = `
<div
  className="makeStyles-template-1"
>
  <Header />
  <div />
  <Footer />
</div>
`;
